src/main/java/org/minions/devfund/chelo/MovieDatabase.java

```diff
@@ -0,0 +1,186 @@
+                getActorList().add(newActor);
+            } else {
+                Actor existingActor = getActorList().get(getActorList().indexOf(new Actor(actor)));
+                ArrayList<Movie> existingMovies = existingActor.getMovies();
+
+                if (!existingActor.getMovies().contains(new Movie(name))) {
+                    existingMovies.add(new Movie(name));
+                }
+                existingActor.setMovies(existingMovies);
+
+            }
+        }
+    }
+
+    /**
+     * Add a rating for this movie.
+     * @param name of the movie
+     * @param rating of the movie
+     */
+    void addRating(final String name, final double rating) {
+        for (Movie movie: getMovieList()) {
+            if (movie.getName().contains(name)) {
+                movie.setRating(rating);
+                break;
+            }
+        }
+    }
+
+    /**
+     * Updates the current rating of a movie with this new rating.
+     * @param name of the movie.
+     * @param newRating of the movie.
+     */
+    void updateRating(final String name, final double newRating) {
+        movieList.get(movieList.indexOf(new Movie(name))).setRating(newRating);
+    }
+
+    /**
+     * Returns the name of the actor that has the best average rating for their movies.
+     * @return the name of the best actor.
+     */
+    String getBestActor() {
+        double maxActorRating = 0;
+        String maxActorName = null;
+        for (Actor actor : getActorList()) {
+            if (actor.getActorRating(getMovieList()) >= maxActorRating) {
+                maxActorRating = actor.getActorRating(getMovieList());
+                maxActorName = actor.getName();
+            }
+        }
+        return maxActorName;
+    }
+
+    /**
+     * Returns the name of the movie with the highest rating.
+     * @return best movie.
+     */
+    String getBestMovie() {
+        double maxRatingMovie = 0;
+        String maxRatingMovieName = null;
+        for (Movie movie : getMovieList()) {
+            if (movie.getRating() >= maxRatingMovie) {
+                maxRatingMovie = movie.getRating();
+                maxRatingMovieName = movie.getName();
+            }
+        }
+        return maxRatingMovieName;
+    }
+
+    /**
+     * Main method.
+     * @param args args
+     */
+    public static void main(final String[] args) {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        File fileName = new File("./resources/movies.txt");
+        try {
+            Scanner read = new Scanner(fileName, "UTF-8");
+
+            while (read.hasNextLine()) {
+                String[] actorMovies = read.nextLine().split(",", -1);
+                for (int i = 1; i < actorMovies.length; i++) {
+                    movieDatabase.addMovie(actorMovies[i], new String[]{actorMovies[0] });
+                }
+            }
+            read.close();
+        } catch (FileNotFoundException ex) {
+            System.out.println(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)